science/projects/soroush_heidary/01-main-content/Main-Content.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    X = pd.concat([df_normal.sample(SIZE), df_target], axis=0)
</pre><p>Thank you for the great charts and analyses you provided in the following sections. </p><p> </p><p>I have a suggestion that in addition to training with an equal number of data for each label, you also repeat the training, testing, and analysis with imbalanced data so that differences, especially in the most important feature, can be identified.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>